pinecone/sparse/bm25/bm25-quora.ipynb

```diff
@@ -0,0 +1,2451 @@
+{
```

Comment:
<blockquote>We'll install the required libraries: the <code>pinecone-client</code> for interacting with Pinecone, the <code>pinecone-datasets</code> library that we will use for fast processing of the Quora dataset, and <code>numpy</code>.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>